### PR TITLE
Tag additional entries for web-features

### PR DESCRIPTION
**Pull request by foolip**
These are cases where the tag already exists in BCD, but additional
entries are being tagged. This is to bring tags in sync with
web-features, so that we can remove compat_keys in most cases.

### CODE REVIEW

File: api/CSSContainerRule.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSContainerRule",
         "spec_url": "https://drafts.csswg.org/css-contain-3/#the-csscontainerrule-interface",
+        "tags": [
+          "web-features:container-queries"
```
Comment (foolip) [inline api/CSSContainerRule.json:8]: This is from https://github.com/web-platform-dx/web-features/pull/556

File: api/CSSFontFeatureValuesRule.json [hunk 2]
```diff
@@ -31,7 +31,10 @@
           "experimental": false,
           "standard_track": true,
           "deprecated": false
-        }
+        },
+        "tags": [
+          "web-features:font-variant-alternates"
```
Comment (foolip) [inline api/CSSFontFeatureValuesRule.json:36]: This and the next file is from https://github.com/web-platform-dx/web-features/pull/772

File: css/properties/animation.json [hunk 3]
```diff
@@ -139,7 +139,10 @@
               "experimental": false,
               "standard_track": true,
               "deprecated": false
-            }
+            },
+            "tags": [
+              "web-features:scroll-driven-animations"
```
Comment (foolip) [inline css/properties/animation.json:144]: This (css.properties.animation.animation-timeline_included) was listed in https://github.com/web-platform-dx/web-features/pull/582 but not in https://github.com/mdn/browser-compat-data/pull/22644 and seems like a simple oversight.

File: css/types/length.json [hunk 4]
```diff
@@ -183,7 +183,10 @@
               "experimental": false,
               "standard_track": true,
               "deprecated": false
-            }
+            },
+            "tags": [
+              "web-features:container-queries"
```
Comment (foolip) [inline css/types/length.json:188]: This is from https://github.com/web-platform-dx/web-features/pull/526

File: javascript/builtins/TypedArray.json [hunk 5]
```diff
@@ -137,7 +137,10 @@
               "experimental": false,
               "standard_track": true,
               "deprecated": false
-            }
+            },
+            "tags": [
+              "web-features:array-at"
```
Comment (foolip) [inline javascript/builtins/TypedArray.json:142]: This is from https://github.com/web-platform-dx/web-features/pull/624

File: javascript/operators/super.json [hunk 6]
```diff
@@ -40,7 +40,10 @@
             "experimental": false,
             "standard_track": true,
             "deprecated": false
-          }
+          },
+          "tags": [
+            "web-features:class-syntax"
```
Comment (foolip) [inline javascript/operators/super.json:45]: This is from https://github.com/web-platform-dx/web-features/pull/770.

**Review by foolip — COMMENTED** [review]
Self-review to check I'm not adding something back that was removed from BCD. Couldn't find any such mistakes.

**Review by Elchi3 — APPROVED** [review]
Thanks! 👍

**Pull request merged**